nixos/modules/virtualisation/containers.nix

```diff
@@ -134,6 +134,9 @@ let
         --setenv HOST_ADDRESS6="$HOST_ADDRESS6" \
         --setenv LOCAL_ADDRESS6="$LOCAL_ADDRESS6" \
         --setenv PATH="$PATH" \
+        ${if cfg . "additionalCapabilities" or null != null then
+          ''--capability="${concatStringsSep " " cfg . "additionalCapabilities"}"'' else ""
```

Comment:
What's up with the quotes around attribute names? It should be `cfg.additionalCapabilities`, not `cfg . "additionalCapabilities"`.

The `or null` should not be necessary. The NixOS option system should ensure that `additionalCapabilities` is always defined. If that's not the case, there is an underlying problem that should be fixed, not hacked around.